packages/nc-gui-v2/pages/signup/[[token]].vue

```diff
@@ -152,6 +154,18 @@ function resetError() {
               </span>
             </button>
 
+            <a
+              v-if="appInfo.googleAuthEnabled"
+              href="http://localhost:8080/auth/google"
+              class="submit after:(!bg-white) !text-primary border-1 border-primary"
+            >
+              <span class="flex items-center gap-2">
+                <LogosGoogleGmail />
+
+                Sign up with Google
```

Comment:
Would it be better to display without underline?

![image](https://user-images.githubusercontent.com/35857179/185754095-3af54148-71fc-4b0a-9d20-426b94fdfa35.png)